spec/javascript/components/supplemental_application/SupplementalApplicationPage.test.js

```diff
@@ -138,14 +139,6 @@ describe('SupplementalApplicationPage', () => {
 
   test('it saves a live/work application preference panel', async () => {
     const withValidPreferences = cloneDeep(supplementalApplication)
-    const payload = cloneDeep(mockShortFormSubmitPayload())
```

Comment:
what was the motivation behind removing this piece?

Reply:
that payload was never used in any of the expects, we created and merged it below but we do not use it anywhere else in the test